app/templates/components/navigation-bar.hbs

```diff
@@ -1,5 +1,7 @@
 <div class="container">
-  <a class="navbar-brand" href="index.html">conduit</a>
+  {{#link-to "home" (query-params tag=null page=1) class="navbar-brand" }}
```

Comment:
I've just merged #76 so we'll need to reset another query param. We'll need to also add `feed=null`

Reply:
@patocallaghan yea probably my fault 🙈 I reviewed this and felt too chicken to merge it at the same time but I suppose I should have 👍 

Do you want me to write an issue for it?

Reply:
ah, no biggie. I can fix it in my next PR

Reply:
@patocallaghan yes it would be great if you could just add it to your next PR (to keep the noise down)